order.js

```diff
@@ -0,0 +1,69 @@
+// $('.card-action').on('click', 'a', clearOrder);
+$('.card-action').on('click', 'a', addItem);
+$('.card-action').on('click', 'a', updateTotal);
+
+let $subtotal = $('.subtotal');
+let $tax = $('.tax');
+let $total = $('.total');
+let runningTotal = 0
+
+
+$(document).ready(function() {
+  $(".button-collapse").sideNav();
+});
+
+function addItem(event) {
+  const $row = $('<tr>');
+
+  $row.append(addDetail(event, 'span').addClass('item'));
+  $row.append(addDetail(event, 'p').addClass('right-align price'));
+  $('.striped tbody').append($row);
+}
+
+function addDetail(event, tagType) {
+  const detail = $(event.target).parent().prev().children(tagType).text();
+  const newCell = $('<td>');
+  return newCell.text(detail);
+}
+
+function updateTotal(event) {
+  let price = addDetail(event, 'p').text();
+  price = Number(price.replace('$', ''));
+  runningTotal += price;
+  $subtotal.text(`$${runningTotal.toFixed(2)}`);
+  $tax.text(`$${(runningTotal * 0.1).toFixed(2)}`);
+  $total.text(`$${(runningTotal * 1.1).toFixed(2)}`);
+}
+
+// validation
+  $('.order-button').on('click', validateName);
+  $('.order-button').on('click', validatePhone);
+  $('.order-button').on('click', validateAddress)
+
```

Comment:
The above lines should go to the top of the file along with the immediately invoked code.